docs/VERCEL_DEPLOY.md

```
@@ -0,0 +1,1214 @@
+
+Vercel と Supabase の連携における詳細な設定方法です。
+
+### データベースマイグレーション
+
+#### 自動デプロイ (推奨)
+
+\`supabase/migrations\` ディレクトリ配下の \`.sql\` マイグレーションファイルは、\`main\` ブランチへのマージ時に自動的に Cloud Supabase へデプロイされます。
+
+**GitHub Actions による自動マイグレーション**:
+
+1. PRを作成し、\`supabase/migrations/\*_/_.sql\` に変更を含める
```

markdownlint-cli2 (0.18.1)

Emphasis style
Expected: asterisk; Actual: underscore

(MD049, emphasis-style)